hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestZKConnectionRegistry.java

```diff
@@ -101,8 +116,8 @@ public void testIndependentZKConnections() throws IOException {
       otherConf.set(HConstants.ZOOKEEPER_QUORUM, MiniZooKeeperCluster.HOST);
       try (ZKConnectionRegistry otherRegistry = new ZKConnectionRegistry(otherConf)) {
         ReadOnlyZKClient zk2 = otherRegistry.getZKClient();
-        assertNotSame("Using a different configuration / quorum should result in different " +
-          "backing zk connection.", zk1, zk2);
+        assertNotSame("Using a different configuration / quorum should result in different "
+          + "backing zk connection.", zk1, zk2);
```

Comment:
I recall that you mentioned many times that the operator must be at the end of a line so I changed my formatter config to do this, so now you prefer the operator at the start of a line?
I'm fine with both, just want to know which one do you want, or it is not important?

Reply:
Mistake. I like '+' at end of line. Let me fix.